client/verta/verta/_deployment/endpoint.py

```diff
@@ -128,23 +129,21 @@ def _get_json_by_path(cls, conn, workspace, path):
                 return endpoint
         return None
 
-    def update(self, run, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
-        if not isinstance(run, experimentrun.ExperimentRun):
-            raise TypeError("run must be an ExperimentRun")
+    def update(self, run_or_model_version, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
+        if not (isinstance(run_or_model_version, experimentrun.ExperimentRun) or isinstance(run_or_model_version, RegisteredModelVersion)):
```

Comment:
`isinstance()` supports tuples for its second argument to check against multiple types!
```suggestion
        if not isinstance(run_or_model_version, (experimentrun.ExperimentRun, RegisteredModelVersion)):
```

Reply:
oh wow nice!

Reply:
Fixed.